ci/kokoro/docker/e2e/spanner_e2e.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <grpcpp/grpcpp.h>
+#include "google/spanner/admin/instance/v1/instance_admin.gapic.h"
+
+using ::google::spanner::admin::instance::v1::ListInstanceConfigsRequest;
+
+int main() {
+  auto creds = grpc::GoogleDefaultCredentials();
+  auto stub = CreateInstanceAdminStub(creds);
+  InstanceAdmin client(std::move(stub));
```

Comment:
nit: if the stub is only used once in the next line maybe this:

```C++
InstanceAdmin client(CreateInstanceAdminStub(creds));
```

Reply:
Done